### PR TITLE
Fix issue with Prismic CMS

### PR DESCRIPTION
**Pull request by levimykel**
- Fix issue with public Prismic repository
- Update instructions for Prismic
- Update the Prismic Custom Type model configurations

### CODE REVIEW

File: lib/cms-providers/prismic/custom_types/stage.json [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
-      "type": "Group",
-      "config": {
-        "fields": {
-          "talk": {
-            "type": "Link",
-            "config": {
-              "select": "document",
-              "customtypes": ["talk"],
-              "label": "Talk"
+    "schedule" : {
```
Comment (leerob) [inline lib/cms-providers/prismic/custom_types/stage.json:30]: Where these spaces intentionally added?

File: lib/cms-providers/prismic/README.md [hunk 2]
```diff
@@ -34,6 +34,12 @@ Then update the environment variables with your Prismic repo ID. Your repo id wi
 PRISMIC_REPO_ID=your-repo-name
 ```
 
+Next at the top of the file, make sure to remove the the value from `DATOCMS_READ_ONLY_API_TOKEN`. The project won't source the content from Prismic until this has been done.
```
Comment (leerob) [inline lib/cms-providers/prismic/README.md:37]: One there's a Prismic integration we can remove this file entirely 🙌

File: lib/cms-providers/prismic/custom_types/stage.json (hunk 1 above)
Comment (levimykel) [reply lib/cms-providers/prismic/custom_types/stage.json:30]: The spaces were not an intentional add, it was generated that way when exported from Prismic.

File: lib/cms-providers/prismic/README.md (hunk 2 above)
Comment (levimykel) [reply lib/cms-providers/prismic/README.md:37]: Yes, the plan is to completely remove these instructions when there's a Prismic integration.

**Review by leerob — APPROVED** [review]

**Pull request merged**